frontend/cypress/e2e/agreementList.cy.js

```diff
@@ -6,21 +6,62 @@ beforeEach(() => {
     cy.visit("/agreements");
 });
 
-afterEach(() => {
-    cy.injectAxe();
-    cy.checkA11y(null, null, terminalLog);
-});
+// afterEach(() => {
+//     cy.injectAxe();
+//     cy.checkA11y(null, null, terminalLog);
+// });
```

Comment:
Is this for testing, or not needed?

Reply:
good catch, I was troubleshooting and forgot to revert